azure-pipelines.yml

```diff
@@ -23,9 +23,15 @@ jobs:
     steps:
       - bash: |
           set -euo pipefail
-          echo "##vso[task.setvariable variable=branch;isOutput=true]$(git rev-parse 'HEAD^2')"
-          echo "##vso[task.setvariable variable=master;isOutput=true]$(git rev-parse 'HEAD^1')"
-          echo "##vso[task.setvariable variable=fork_point;isOutput=true]$(git merge-base --fork-point 'HEAD^1' 'HEAD^2')"
+          if [ "$(Build.Reason)" == "PullRequest" ]; then
+              echo "##vso[task.setvariable variable=branch;isOutput=true]$(git rev-parse 'HEAD^2')"
```

Comment:
`HEAD^2` does not exist for master commits as they have a single parent (squash-merge).